### PR TITLE
Eclipse 2022-09 support

### PR DESCRIPTION
**Pull request by kolipakakondal**
## Description

Unable to install IDF eclipse plugin on Eclipse 2022-09 CDT due to the following reasons:
1. Latest Eclipse CDT is missing with the org.apache.batik.xml plugin
2. Old embedded CDT plugins have become invalid with the latest CDT

What did we do?
1. Added org.apache.xml.batik  to the idf update site plugins update
2. Updated swtcharts from 0.12.0 to 0.1.3.0
3. Updated embedded CDT core plugins to the latest
4. Removed unwanted embedded plugins from the target definition

Fixes # ([IEP-768](https://jira.espressif.com:8443/browse/IEP-768))

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Testes which are required?

Test:1
- Install Eclipse CDT 2022-09
- Install PR using the update site 

Test:2
- Install Embedded CDT 2022-09
- Install PR using the update site 

Test:3
- Install Espressif-IDE v2.6.0
- Install PR using the update site

Test:4
- Install Espressif-IDE v2.6.0
- Install PR using the update site
- Verify features which impacted with this - application size analysis, OpenOCD debugging 

**Test Configuration**:
* ESP-IDF Version: master
* OS (Windows,Linux and macOS): All

## Dependent components impacted by this PR:

- IEP plugin update site
- Application size analysis
- OpenOCD debugging
- Eclispe 2022-09 is a new release, so it would be great if we could test all basic scenarios with this PR

## Checklist
- [ ] PR Self Reviewed
- [ ] Applied Code formatting
- [ ] Added Documentation
- [ ] Added Unit Test
- [ ] Verified on all platforms - Windows,Linux and macOS

### CODE REVIEW

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeDataManager.java [hunk 1]
```diff
@@ -184,8 +182,25 @@ protected List<String> getCommandArgsArchives(String pythonExecutablenPath, IFil
...
-		arguments.add("--json"); //$NON-NLS-1$
+		arguments.addAll(addJsonParseCommand());
+
+		return arguments;
+	}
 
+	private List<String> addJsonParseCommand()
+	{
+		List<String> arguments = new ArrayList<String>();
+		IEnvironmentVariable idfVersionEnv = new IDFEnvironmentVariables().getEnv("ESP_IDF_VERSION"); //$NON-NLS-1$
```
Comment (kolipakakondal) [inline bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeDataManager.java:193]: Hi @sigmaaa Are we updating `ESP_IDF_VERSION` with esp-idf installation and configuration?

Comment (sigmaaa) [reply bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeDataManager.java:193]: Hi @kolipakakondal. No, we are not updating it. I'll fix that. Thanks

Comment (sigmaaa) [reply bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeDataManager.java]: fixed

**Comment by AndriiFilippov** [comment]
@kolipakakondal

Tested under:
OS - Linux
ESP-IDF: master

LGTM

**Comment by kolipakakondal** [comment]
> @kolipakakondal
> 
> Tested under: OS - Windows 10 ESP-IDF: master
> 
> **Test:1** Install Eclipse CDT 2022-09 Install PR using the update site Create build project -> Check "application size analysis" -> Check "Debugging" Install ESP-IDF Eclipse Plugin -> Create build project -> Check "application size analysis" -> Check "Debugging"
> 
> **Test:2** Install Embedded CDT 2022-09 Install PR using the update site Create build project -> Check "application size analysis" -> Check "Debugging" Install ESP-IDF Eclipse Plugin -> Create build project -> Check "application size analysis" -> Check "Debugging"
> 
> While installing ESP-IDF Eclipse Plugin -> get this notification only: image
> 
> LGTM. Continue testing with Linux

I think this should be fine considering it's trying to update to the latest batik plugin since we bundled this plugin along with the idf update site.

Thanks @AndriiFilippov for verifying this

**Pull request merged**